### DISCUSSION TITLE
Installation Wiki.js | delete from "locales" where "code" != $1

### DISCUSSION THREAD

**Discussion by fallword** [requarks/wiki#6092] [Error / Bug Report]
Hello,

When I try to install the wiki.js tool via the web interface "http:\IP:3000", I fill in
the different fields. But as soon as I press the "Install" button, it gives me the
page for the installation of the product. As soon as I fill in the different fields a second time, a loading takes place.

Then it displays an error "delete from "locales" where "code" != $1 - update or delete on table "locales" violates foreign key constraint "users_localecode_foreign" on table "users""

And after that, I can access the wiki.js login interface however, I would like to know why this is displayed? Is it normal?

However, when I disable the firewall of the vm (iptables) the installation proceeds correctly.

I enclose the vm firewall rules:

!/bin/sh

iptables -t filter -F
iptables -t filter -X

iptables -t filter -P INPUT DROP
iptables -t filter -P FORWARD DROP
iptables -t filter -P OUTPUT DROP

iptables -A INPUT -m state --state RELATED,ESTABLISHED -j ACCEPT
iptables -A OUTPUT -m state --state RELATED,ESTABLISHED -j ACCEPT

iptables -t filter -A INPUT -i lo -j ACCEPT
iptables -t filter -A OUTPUT -o lo -j ACCEPT

iptables -t filter -A INPUT -p icmp -j ACCEPT
iptables -t filter -A OUTPUT -p icmp -j ACCEPT

iptables -t filter -A INPUT -p tcp --dport 4222 -j ACCEPT
iptables -t filter -A OUTPUT -p tcp --dport 4222 -j ACCEPT

iptables -t filter -A INPUT -p tcp --dport 3000 -j ACCEPT
iptables -t filter -A OUTPUT -p tcp --dport 3000 -j ACCEPT

iptables -t filter -A OUTPUT -p tcp --dport 53 -j ACCEPT
iptables -t filter -A OUTPUT -p udp --dport 53 -j ACCEPT
iptables -t filter -A INPUT -p tcp --dport 53 -j ACCEPT
iptables -t filter -A INPUT -p udp --dport 53 -j ACCEPT

iptables -t filter -A OUTPUT -p udp --dport 123 -j ACCEPT

I am sending you the different information of the products used for the installation of wiki.js

OS : Ubuntu 22.04 (Server)

Database : PostgreSQL
- Version = 14.6

Node.JS : Version = V16.19.0

I thank you in advance for the help you can give me !

**Comment by NGPixel** [comment]
Please don't create duplicate issues.

**Reply by fallword** [reply]
Hello,

I apologize for this duplication.

I wanted to move my post to the "Report a Bug" category.

I wish you an excellent day.
